src/.vuepress/components/common/codepen-snippet.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <p
+    class="codepen"
+    data-theme-id="39028"
+    data-preview="true"
+    data-editable="true"
```

Comment:
Some of the current examples are not editable. Specifically, those in the Examples section.

Is changing this intentional?

Reply:
Initially I thought we could define a standard to it, but after more thinking I realized that each section can have its own reasons to allow or not inline editable examples. So, my intention is not to change any current behavior, just improve future maintenance with the component. My recent commits added this behavior as props too.